src/utils.js

```diff
@@ -22,67 +31,103 @@ const getAvatarAndName = () => {
   });
 };
 
-const saveStateEvent = (e, text, eventToEdit, eventsList, setEventList) => {
+const getEventData = (eventId) => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const events = JSON.parse(localStorage.getItem("Events"));
+      const event = events.find((event) => event.id === eventId);
+
+      if (!event) {
+        console.log("No event found");
+        reject(null);
+      } else {
+        resolve({ event });
+      }
+    }, 500);
+  });
+};
+
+const saveStateEvent = (e, text, eventsList, setEventList) => {
```

Comment:
I think we are going to remove this function ( `saveStateEvent` ) , since we are not going to use the `stateEvent` field any more